tests/matrix_free/parallel_multigrid_adaptive_06.with_mpi=true.with_p4est=true.mpirun=7.output

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
Do we really need two variants of this test? I think it would be enough to have the test output with 3 ranks and delete this one. The reason we had 7 ranks in the original test is that there were some corner cases where the transfer missed some weights in the preliminary implementation, but here we simply want to see that the block vector gets the right partitions etc which we should also have with 3 ranks, shouldn't we?